CMakeLists.txt

```diff
@@ -56,6 +56,10 @@ if(MOLD_USE_MOLD)
   add_link_options(-fuse-ld=mold -Wl,--gdb-index)
 endif()
 
+if(MINGW)
+  set(CMAKE_DL_LIBS -ldl)
```

Comment:
If this is equivalent to `target_link_libraries(mold PRIVATE dl)`, please do so instead of overriding the system-provided variable.

Reply:
Done